frappe/patches/v12_0/update_gender_and_salutations.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+
+from __future__ import unicode_literals
+import frappe
+from frappe.desk.page.setup_wizard.install_fixtures import update_genders, update_salutations
+
+def execute():
+	frappe.db.set_value("DocType", "Contact", "module", "Contacts")
+	frappe.db.set_value("DocType", "Address", "module", "Contacts")
+	frappe.db.set_value("DocType", "Address Template", "module", "Contacts")
+	frappe.reload_doc('contacts', 'doctype', 'gender')
```

Comment:
`frappe.reload_doc` is not required as you are not changing json of gender or salutation